src/sinks/humio/metrics.rs

```diff
@@ -293,4 +293,47 @@ mod tests {
             output[1].1
         );
     }
+
+    #[tokio::test]
+    async fn duplicate_metrics() {
```

Comment:
Isn't this to test duplicate tags?

Also, should we converge on calling it multi-value tags or duplicate tags?

Reply:
I've been avoiding the term "duplicate" as the values are not duplicated, only the names.